Dockerfile

```diff
@@ -24,16 +24,16 @@ WORKDIR /tmp
 ENV DEBIAN_FRONTEND=noninteractive
 
 # Tooling
-ARG SUBWASM_VERSION=0.18.0
+ARG SUBWASM_VERSION=0.19.0
 ARG TERA_CLI_VERSION=0.2.2
 ARG TOML_CLI_VERSION=0.2.2
 
 COPY ./templates ${SRTOOL_TEMPLATES}/
 RUN apt update && \
     apt upgrade -y && \
     apt install --no-install-recommends -y \
-        cmake pkg-config libssl-dev make protobuf-compiler \
-        git clang bsdmainutils ca-certificates curl && \
```

Comment:
Please keep this indentation, those are not commands but the args to the command we can spot above when the indentation is preserved.

Reply:
Did that in https://github.com/paritytech/srtool/pull/60/commits/2372b606a9ccd280df03113ef176355459089ae9. My IDE automatically formatted that in what it thinks was the proper way.

Reply:
Please don't let your IDE do its magic, what it does does not help and the indenting was previously correct.

Reply:
Is it still not correct now? I meant that what I pushed was the resulting of my IDE formatting. I reverted the change based on your suggestion below. Do I also have to revert the second line that was changed?